src/executables/CMakeLists.txt

```diff
@@ -178,6 +178,13 @@ endif()
 
 add_amanzi_executable(ats
   SOURCE main.cc
-  LINK_LIBS ats_executable ${fates_link_libs} ${tpl_link_libs} ${ats_link_libs} ${amanzi_link_libs} 
+  LINK_LIBS ats_executable ${fates_link_libs} ${tpl_link_libs} ${ats_link_libs} ${amanzi_link_libs}
   OUTPUT_NAME ats
   OUTPUT_DIRECTORY ${ATS_BINARY_DIR})
+
+# set this here for now
+# add option to bootstrap before completion
+if (BUILD_ELM_ATS_API)
```

Comment:
this probably needs to be updated to ENABLE_ELM_ATS_API now, otherwise this PR is ready to go.

Reply:
Done with c330f4f